docs/docs/09-typescript/01-resources.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Resources
+id: resources
+---
+
```

Comment:
Did you consider interleaving typescript and winglang in the root documentation (maybe with language tabs like other frameworks). Seems like there are a bunch of topics here that might be duplicated and it will be hard to maintain.

Reply:
I did but I struggled to find a way to do it that didn't make them worse together. I think we should eventually combine them but for now the overall isn't really that much, and in most cases I simply add a link to other pages for more info.